.readthedocs.yaml

```diff
@@ -28,7 +28,7 @@ build:
     pre_install:
       - git update-index --assume-unchanged docs/rtd_environment.yaml docs/conf.py
     post_install:
-      - towncrier build --keep
+      - git describe --exact-match || towncrier build --keep
```

Comment:
How do we prove that this works? Would a proof-of-concept draft PR with the same change but against https://github.com/spacetelescope/jwst/tree/release/1.19.x prove the point if RTD is green for that PR?

Reply:
One way to reproduce the error locally is to checkout the current release branch and try to run `towncrier build --keep` after syncing tags (this replicates what rtd is doing). If I run that it fails with the same error:
```
ValueError: It seems you've already produced newsfiles for this version?
```
See https://app.readthedocs.org/projects/jwst-pipeline/builds/29547492/
On the same commit running `git describe --exact-match || towncrier build --keep` return "1.19.1" (and doesn't re-render the already rendered changelog.

I can't think of a way to test this in a PR since it requires the commit be tagged.